resources/dicts/thoughts/dead/starclan/deputy.json

```diff
@@ -0,0 +1,11 @@
+[
+    {
+        "id": "deputy_dead_deputy",
+        "thoughts": [
+            "Wishes they could speak to r_c to learn how to assign patrols better"
```

Comment:
A dead cat is thinking this? --> duplicate in another deputy thought (maybe check on that)